src/ast/CMakeLists.txt

```diff
@@ -23,7 +23,39 @@ target_link_libraries(ast arch)
 add_dependencies(ast parser)
 
 if (STATIC_LINKING)
-  set(clang_libs
+  set(all_clang_libs
```

Comment:
I think that you can just use CLANG_EXPORTED_TARGETS as that seems to be the public interface of clang's CMake file

Reply:
Ah, good tip. I'm getting link errors but will play around with it.

Reply:
The link errors seem related to the semicolon join. If I leave the join in place and strip LLVM from everything in CLANG_EXPORTED_TARGETS, the binary links and works.

This VM is running Ubuntu 19.10

```
$ cmake --version
cmake version 3.13.4
```

Perhaps we need to gate the semicolon join behind a cmake version check?

Reply:
Semicolons are interpolated when you quote them I think, it's a weird Cmake quirk. Try passing the list unquoted to see if CMake handles it better. This semicolon quirk of cmake seems to be because semicolons are used by cmake's list representation.